pkg/registry/core/namespace/storage/storage_test.go

```diff
@@ -188,7 +188,7 @@ func TestUpdateDeletingNamespaceWithIncompleteMetadataFinalizers(t *testing.T) {
 	defer server.Terminate(t)
 	defer storage.store.DestroyFunc()
 	key := "namespaces/foo"
-	ctx := genericapirequest.NewContext()
+	ctx := genericapirequest.WithNamespace(genericapirequest.NewContext(), metav1.NamespaceNone)
```

Comment:
namespace is set in the context based on whether there's a `namespaces/$namespace` in the request URL. `Namespace` delete requests set the namespace name in the context.